src/test/java/com/marklogic/kafka/connect/source/AbstractIntegrationSourceTest.java

```diff
@@ -31,6 +31,21 @@ public class AbstractIntegrationSourceTest extends AbstractIntegrationTest {
     protected final String AUTHORS_OPTIC_DSL = "op.fromView(\"Medical\", \"Authors\")";
     protected final String AUTHORS_OPTIC_SERIALIZED = "{\"$optic\":{\"ns\":\"op\", \"fn\":\"operators\", \"args\":[{\"ns\":\"op\", \"fn\":\"from-view\", \"args\":[\"Medical\", \"Authors\"]}]}}";
     protected final String AUTHORS_TOPIC = "Authors";
+    protected final String JSON_RESULT = "{\"Medical.Authors.ID\":{\"type\":\"xs:integer\",\"value\":2}," +
+        "\"Medical.Authors.LastName\":{\"type\":\"xs:string\",\"value\":\"Pulhoster\"}," +
+        "\"Medical.Authors.ForeName\":{\"type\":\"xs:string\",\"value\":\"Misty\"}}";
+    protected final String XML_RESULT = "<Record><t:columns xmlns:t=\"http://marklogic.com/table\">\n" +
```

Comment:
Just define the XML and CSV results in the classes that need them, which likely will only be those two classes. JSON result is fine to have here, although I'd probably stuff it in ReadRowsViaOpticDslTest as a public static and have the serialized test reference it there. That helps keep this parent class a little more lean.